doc/content/concepts/devicetemplates/assigning-euis.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Assigning EUIs"
+description: ""
+weight: 6
+---
+
+You can use the `device templates assign-euis` command to assign the `JoinEUI` and `DevEUI` to device templates. The `DevEUI` is assigned from the given start number. This personalizes the device templates.
```

Comment:
Shouldn't we make clear that this is a CLI command? either by prefixing `ttn-lw-cli` or by adding "of the CLI"